src/genie/libs/parser/ironware/show_mpls.py

```diff
@@ -0,0 +1,691 @@
+"""
+Module:
+    genie.libs.parser.ironware.show_mpls
+
+Author:
+    James Di Trapani <james@ditrapani.com.au> - https://github.com/jamesditrapani
+
+Description:
+    MPLS parsers for IronWare devices
+
+Parsers:
+    * show mpls lsp
+    * show mpls vll <vll>
+    * show mpls vll-local <vll>
+    * show mpls ldp neighbor
+"""
+
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Or, Optional
+import re
+
+__author__ = 'James Di Trapani <james@ditrapani.com.au>'
+
+
+# ======================================================
+# Schema for 'show mpls lsp wide'
+# ======================================================
+class ShowMPLSLSPSchema(MetaParser):
+    """Schema for show mpls lsp"""
+    schema = {
+        'lsps': {
+            Any(): {
+                'destination': str,
+                'admin': str,
+                'operational': str,
+                'flap_count': int,
+                'retry_count': int,
+                Optional('tunnel_interface'): str,
+                Optional('path'): str
+            }
+        }
+    }
+
+
+# ====================================================
+#  parser for 'show mpls lsp wide'
```

Comment:
@KamyarZiabari Brocade/Ironware has no concept of `terminal width 0` like Cisco. As such, wide needs to be appended to commands to prevent them being compressed. Here is an example:

`show mpls lsp`:
```
ssh@edge1.code1.city#sh mpls lsp
Note: LSPs marked with * are taking a Secondary Path
                               Admin Oper  Tunnel   Up/Dn Retry Active
Name           To              State State Intf     Times No.   Path
edge1.code1   192.168.1.2  UP    UP    tnl3     2     0     --
.city_to_lsr1
.code2.city
edge1.code1   192.168.1.137  UP    UP    tnl0     2     0     --
.city_to_lsr3
.code3.city
edge1.code1   192.168.1.24   UP    UP    tnl2     2     0     --
.city_to_lsr3
.code4.city
edge1.code1   192.168.1.138  UP    UP    tnl1     1     0     --
.city_to_lsr5
.code5.city
```

`show mpls lsp wide`:
```
ssh@edge1.code1.city#sh mpls lsp wide
Note: LSPs marked with * are taking a Secondary Path
                                                                                 Admin Oper  Tunnel   Up/Dn Retry Active
Name                                                             To              State State Intf     Times No.   Path
edge1.code1.city_to_lsr1.code2.city                              192.168.1.2  UP    UP    tnl3     2     0     --                    
edge1.code1.city_to_lsr3.code3.city                              192.168.1.137  UP    UP    tnl0     2     0     --                    
edge1.code1.city_to_lsr3.code4.city                              192.168.1.24   UP    UP    tnl2     2     0     --                    
edge1.code1.city_to_lsr5.code5.city                              192.168.1.138  UP    UP    tnl1     1     0     --
```

Let me know if this is okay.